src/coreclr/jit/flowgraph.cpp

```diff
@@ -4047,8 +4049,9 @@ void Compiler::fgSetBlockOrder()
     {
         printf("The biggest BB has %4u tree nodes\n", BasicBlock::s_nMaxTrees);
     }
-    fgDebugCheckLinks();
 #endif // DEBUG
+
+    return PhaseStatus::MODIFIED_NOTHING;
```

Comment:
Shouldn't this be `MODIFIED_EVERYTHING`?

Reply:
Sure. Currently dumping and checking are tied together. I was trying to avoid redundant seeming dumps, but I suppose we want the checks.